components/Box.tsx

```diff
@@ -0,0 +1,3 @@
+import { styled } from '../stitches.config'
+
+export const Box = styled('div', {})
```

Comment:
What's the reason for defining these in this project, rather than importing them from the components package?

Reply:
They should eventually come from `@atom-learning/components`, but I wanted to keep it all local until we have it ready